.github/workflows/legacy-checkpoints.yml

```diff
@@ -1,22 +1,28 @@
 name: Create Legacy Ckpts
 
-# https://help.github.com/en/actions/reference/events-that-trigger-workflows
 on:
   workflow_dispatch:
+    inputs:
+      push_to_s3:
+        description: 'Push generated checkpoints to S3'
+        required: false
+        type: boolean
 
 jobs:
   create-legacy-ckpts:
     runs-on: ubuntu-20.04
+    defaults:
+      run:
+        working-directory: ./legacy
     steps:
     - uses: actions/checkout@v2
 
     - uses: actions/setup-python@v2
       with:
-        python-version: 3.9
+        python-version: 3.8
```

Comment:
Downgraded to 3.8 to cover all PL versions from 1.0.0 to 1.5.10. (e.g. 3.9 is not supported by PL 1.0.0.)